### PR TITLE
[Dependency:#4002] Makes wicker doll and prison cube available via uplink

### PR DESCRIPTION
**Pull request by Rukofamicom**
## About The Pull Request

Adds the wicker (voodoo) doll to Curator and Stage Magician uplinks for 12 TC.
Adds the Prison Cube to Curator uplinks for 6 TC.

image

For those who are unfamiliar with the items: They are both currently available to shaft miners as random drops on lavaland, but they rarely see use due to their antagonistic nature and also weak overall power compared to other rewards. They're much better off as quirky antag-only items for use by players who aren't afraid to sacrifice their livelihood for a little chaotic fun. 

## Why It's Good For The Game

Because these are fun and quirky items that shouldn't be removed entirely from the game per #4002 
Both items are a good match for the Curator's desire to collect oddities, and the Wicker Doll also fits the wizard-lite theme of Stage magician.

## Changelog
:cl:
add: Wicker Doll (Voodoo doll) is now available in traitor uplinks for both Curators and Stage Magicians at a cost of 12 TC
add: Prison cube is now available in traitor uplinks for Curators at a cost of 6 TC
/:cl:

### CODE REVIEW

**Comment by Rukofamicom** [comment]
## This should not be merged until/unless #4002 is merged.

**Comment by PowerfulBacon** [comment]
The prison cube in uplink is ehhhh and the wicker doll costing 12 TC is pretty bruh since its a bit crap

**Comment by Rukofamicom** [comment]
> wicker doll costing 12 TC is pretty bruh since its a bit crap

It's bad for direct combat and very *very* good for sneaky types that don't overuse it and give themselves up. You can't easily kill someone with a voodoo doll, but you can relatively easily cause someone to get killed by a third party. The voodoo doll also allows you to remotely spy on them via eyes targetting. 

> The prison cube in uplink is ehhhh

Prison Cube can only be used on cuffed or sleeping targets, so it's only useful if you have already effectively won the fight and just want to dab on someone with a semi-magical artifact. It has way more downsides than upsides, which is why it's *relatively* cheap. 

If you're referring to it being eh because it's a magical-oriented item, I don't really have much to explain it away lol

***

This PR exists purely to make sure they aren't almost entirely removed from the game. I don't expect we'll see either very often regardless, but we never see them very much currently either. I don't have any hard stances against tweaking TC costs so feel free to tweak at will - I just wanted to explain why I chose the costs I did.

**Comment by DatBoiTim** [comment]
> ## This should not be merged until/unless #4002 is merged.

Note a dependency in title

**Comment by DatBoiTim** [comment]
Anyways extremely based.

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -2058,6 +2058,20 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
+	item = /obj/item/voodoo
+	cost = 12
+	restricted_roles = list("Curator", "Stage Magician")
+
+/datum/uplink_item/role_restricted/prison_cube
+	name = "Prison Cube"
+	desc = "A very strange artifact recovered from a volcanic planet that is useful for keeping people locked away, but not very useful for keeping their disappearance unknown"
+	item = /obj/item/prisoncube
+	cost = 6
+	restricted_roles = list("Curator")
```
Comment (DatBoiTim) [inline code/modules/uplink/uplink_items.dm:2073]: Is there a reason this is not also stage magician available? Considering it's also a semi-magical item.

**Comment by r1ks-iwnl** [comment]
> 
> 
> its a bit crap

I'm pretty sure it can make people use emotes such as surrender to stun.

**Comment by utilbird** [comment]
> ## This should not be merged until/unless #4002 is merged.

You're not my supervisor

**Review by utilbird — APPROVED** [review]

**Comment by DatBoiTim** [comment]
> You're not my supervisor

I laughed at this more than I should have

**Comment by RDS88-beeman** [comment]
unironically want this even if #4002 doesn't get done it would be sick

**Comment by Rukofamicom** [comment]
I mean, I suppose there's *not really* any harm in enabling them while they're still available on lavaland.

🤔

**Pull request merged**